package.json

```diff
@@ -1,6 +1,24 @@
 {
   "name": "homeday-blocks",
   "version": "2.5.0",
+  "description": "A Vue component library built by Homeday's frontend team.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/homeday-de/homeday-blocks.git"
+  },
+  "keywords": [
+    "vue",
+    "javascript",
+    "component library",
+    "storbybook",
```

Comment:
storybook?

Reply:
Oh my, is like I was :beers: . Thanks for extra pair of :eye: .